packages/@aws-cdk/aws-elasticloadbalancingv2/lib/shared/load-balancer-targets.ts

```diff
@@ -111,3 +111,42 @@ export class IpTarget implements IApplicationLoadBalancerTarget, INetworkLoadBal
     };
   }
 }
+
+export class LambdaTarget implements IApplicationLoadBalancerTarget {
```

Comment:
I'm wondering whether this class should go into a separate package, so we can set up dependencies properly so that...

Reply:
A separate package inside `aws-elasticloadbalancingv2`? Like inside `lib/shared`?
I thought about that, but then `IpTarget` and `InstanceTarget` were also here. 
I like the idea and can get going on this in the next few days, if you think it is the way forward.